website/pages/en/community.js

```diff
@@ -103,7 +123,7 @@ function ImplementationsShowcase() {
 function Implementations(props) {
 	const { config: siteConfig } = props;
 	const { repoUrl } = siteConfig;
-	const editUrl = `https://www.finos.org/get-involved-fdc3`;
+	const editUrl = `/get-involved`;
```

Comment:
This page doesn't actually contain details of how to add your implementation to the page.

The original solution was this link (which will let you open a PR to add your details): https://github.com/finos/FDC3/edit/master/website/data/community.json

Either we'll need to update the get involved page, revert to the link, or drop in a mailto: link with a template asking for the relevant details.

Reply:
I'll revert to the link